README.md

```diff
@@ -35,12 +38,30 @@ A project is configured using a simple json like that
 }
 ```
 
+- For `freelancer` mode: 
+  * not implemented
+
+- For `team` mode:
+  * not implemented
```

Comment:
That is not so clear at moment I think that can be removed by now and introduced the concept in a next PR

Reply:
Please if agree with that remove the empty models well

Reply:
I agree, we can remove for now and add when implemented